### ISSUE TITLE
Ratiogroup question

### ISSUE THREAD

**Issue by xtendpayaa** [rakshasa/rtorrent#965]
Hi there!
How exactly the ratiogrup works in here?
I mean for example in rutorrent there is the ratiogroup settings:
there is ratio no., name, min%, max%, ul (MiB), time (hour), action
it is clear but there is OR/OR or AND/OR?
I mean I set the ratio min% max% 100 and put 48 hours and action delete.
But its doesn't matter if its reach 100% upload or not when 48 hours ended then rtorrent delete the torrent from the client.
I want that IF 100% upload reach AND 48 hours done then delete only. Not OR 100% OR 48 hrs.

**Comment by piramiday** [comment]
you can get more details of the way **rutorrent** does things here:
https://plaza.quickbox.io/t/rutorrent-ratio-groups/449

in **rtorrent** there is no easy way to allow for time-based seeding rules.
you can implement ratio-based seeding rules, though:
https://github.com/rakshasa/rtorrent/wiki/RTorrentRatioHandling

**Comment by pyroscope** [comment]
> in rtorrent there is no easy way to allow for time-based seeding rules.

Wrong. There is rtcontrol for one, which can do the above rules even on a bad day, and if you want to stay "pure" just add a schedule that uses an elapsed.greater condition.

**Comment by piramiday** [comment]
well, I wrote _no easy way_: **easy** being the crucial word, here.
indeed, I don't think that someone who asks about rutorrent ratio groups in the rtorrent github page is extremely comfortable with rtorrent-ps -- I know I'm not (does this `rtcontrol` work with a vanilla `rtorrent` install? does it require `rtorrent-ps`? etc.)

**Comment by bjeanes** [comment]
> does this `rtcontrol` work with a vanilla `rtorrent` install

yes!
